xAPI.md

```diff
@@ -3768,7 +3795,9 @@ allow other uses to emerge.
 
 Example endpoint: http://example.com/xAPI/about
 
-Returns: ```200 OK```, Single 'about' JSON document.
+**Content:** None.
+
+**Returns:** ```200 OK```, Single 'about' JSON document.
```

Comment:
"single" feels weird, as does using document here. "JSON object with basic metadata about this LRS"?